server/src/handlers/posts/delete-post.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express'
+import { Db } from 'mongodb'
+import { ObjectId } from 'mongodb'
+
+/** finds and returns a post by id */
+export const deletePostHandler = (database: Db) => async (
+  req: Request,
+  res: Response,
+) => {
+  const { postId } = req.params
+
+  try {
+    const getPostById = await database
+      .collection('posts')
+      .findOneAndDelete({ _id: new ObjectId(postId) })
+
+    console.log(getPostById)
+
+    return res.status(200).json({
+      updatedPost: getPostById,
```

Comment:
What is sent to client here?

Reply:
Post in json format

Reply:
No need to return deleted pod data, use `.deleteOne` instead